Add new systemd integration (#2868)
DataDog/integrations-core: systemd/README.md

# Agent Check: Systemd

## Overview

This check monitors [Systemd][1].

## Setup

### Installation

> The Systemd check is included in the [Datadog Agent][2] package, so you do not

Reviewer: ```suggestion
The Systemd check is included in the [Datadog Agent][2] package. No additional installation is needed on your server.
```

> need to install anything else on your server.

Reviewer: ```suggestion

```

### Configuration

1. Edit the `systemd.d/conf.yaml` file, in the `conf.d/` folder at the root of your
   Agent's configuration directory to start collecting your systemd performance data.
   See the [sample systemd.d/conf.yaml][2] for all available configuration options.

> 2. [Restart the Agent][3]

Reviewer: ```suggestion
2. [Restart the Agent][3].
```

### Validation

> [Run the Agent's `status` subcommand][4] and look for `systemd` under the Checks section.

Reviewer: ```suggestion
[Run the Agent's status subcommand][4] and look for `systemd` under the Checks section.
```

## Data Collected

### Metrics

> Systemd does not include any metrics.

Reviewer: ```suggestion
Systemd does not include any metrics at this time.
```

### Service Checks

> Systemd does not include any service checks.

Reviewer: ```suggestion
Systemd does not include any service checks at this time.
```

### Events

> Systemd does not include any events.

Reviewer: ```suggestion
Systemd does not include any events at this time.
```

# Agent Check: Systemd

## Overview

This check monitors [Systemd][1] through the Datadog Agent.

## Setup

### Installation

> The Systemd check is included in the [Datadog Agent][2] package. No additional installation is needed on your server.

Reviewer 2: Any reasons not to ship it?

### Configuration

1. Edit the `systemd.d/conf.yaml` file, in the `conf.d/` folder at the root of your
   Agent's configuration directory to start collecting your systemd performance data.
   See the [sample systemd.d/conf.yaml][2] for all available configuration options.

2. [Restart the Agent][3].

### Validation

[Run the Agent's status subcommand][4] and look for `systemd` under the Checks section.

## Data Collected

### Metrics

See [metadata.csv][5] for a list of metrics provided by this check.

### Service Checks

**systemd.unit.active**:  

> Returns `CRITICAL` if the unit's state is 'active' or 'failed', `WARN` if it's 'deactivating', otherwise returns `OK`.

Author: Correct, will need to update that

### Events

The Systemd check emits an event to Datadog each time a unit in the configuration file changes state.

## Troubleshooting

Need help? Contact [Datadog support][5].

> [1]: **LINK_TO_INTEGERATION_SITE**

Reviewer 3: Link is missing here
